README.md

```diff
@@ -143,8 +143,9 @@ git submodule update --init --recursive
 OpenConsole.sln may be built from within Visual Studio or from the command-line using MSBuild. To build from the command line:
 
 ```shell
-.\tools\razzle.cmd
-bcz
+cd tools
+razzle.cmd
+bcz.cmd
```

Comment:
there's also `bz.cmd` which seems to do an incremental build, whereas `bcz.cmd` starts with a clean slate. We should also mention that the AppX bundle isn't built unless you specify a Release build with `bcz.cmd rel` or 'bz.cmd rel`.

Reply:
Changed in accordance to @zadjii-msft comment.